### ISSUE TITLE
Bug: Upgrading openshift from  4.8.13 to 4.8.24 was blocked by OADP 0.2.6

### ISSUE THREAD

**Issue by Frank51** [openshift/oadp-operator#559]
### Contact Details

yunqian.chen@dell.com

### Describe bug

Found out that when upgrading the Openshift Container Platform from 4.8, the upgrade process was blocked since OADP 0.2.6 being incompatible with the newer OCP 4.8.24.

"Cluster operator operator-lifecycle-manager cannot be upgraded between minor versions: ClusterServiceVersioins blocking cluster upgrade: oadp 0.2.6 is incompatible with Openshift minor versions greater than 4.8"

![image](https://user-images.githubusercontent.com/26369715/151603831-54357fcd-f7ba-4559-b8ee-2c999d5df1de.png)

### What happened?

Upgrading the Openshift Container Platform 4.8 failed since OADP 0.2.6.

### OADP Version

0.2.x (Alpha)

### New issue

- [X] This issue is new

**Comment by kaovilai** [comment]
There was an openshift bug around this..
https://bugzilla.redhat.com/show_bug.cgi?id=1994110

**Comment by dymurray** [comment]
@Frank51 Please see my response in slack as well. I believe this message is just telling you you cannot upgrade to 4.9 (the next minor version). z-stream update is available according to the screenshot. Please let me know if you are seeing otherwise.

**Comment by kaovilai** [comment]
You can proceed to update within the 4.8 Channel with 0.2.6 operator.

**Comment by kaovilai** [comment]
> The cluster should not be updated to the next **_minor_** version

The warning is only there to say you cannot upgrade Y (minor version) from 4.8 to 4.9.
Version format follows X.Y.Z per semver.org
ie. for 4.8.13
X - Major: 4 (cannot be greater than 4)
Y - **_Minor_**: 8 (cannot be greater than 8)
Z - Patch: 13 (can be any number above current)
